js/apps/system/_admin/aardvark/APP/react/src/views/databases/addDatabase/AddDatabaseModal.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, Flex, Heading, Stack } from "@chakra-ui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import { mutate } from "swr";
+import * as Yup from "yup";
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader
+} from "../../../components/modal";
+import { getCurrentDB } from "../../../utils/arangoClient";
+import { AddDatabaseForm } from "./AddDatabaseForm";
+import { DatabaseDescription } from "../Database.types";
+import { useFetchInitialValues } from "../useFetchInitialValues";
+
+export const AddDatabaseModal = ({
+  isOpen,
+  onClose
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+}) => {
+  const initialFocusRef = React.useRef<HTMLInputElement>(null);
```

Comment:
We don't need the `initialFocusRef` here, as the default behaviour for the Modal is to [focus on the first tabbable element](https://v1.chakra-ui.com/docs/components/overlay/modal#usage). For Analyzers, that element was different from the "name" input, hence it was required there. 

We can remove usage of `initialFocusRef` completely, and it will work as expected.